<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotterBox</class>
 <widget class="QWidget" name="PlotterBox">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splMngViewBox">
     <property name="lineWidth">
      <number>-7</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="opaqueResize">
      <bool>true</bool>
     </property>
     <widget class="QGroupBox" name="groupPlotBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>270</width>
        <height>40</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QPushButton" name="pushBPrint">
       <property name="geometry">
        <rect>
         <x>204</x>
         <y>6</y>
         <width>58</width>
         <height>26</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>58</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>58</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string>Print</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxPDF">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>3</y>
         <width>40</width>
         <height>17</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>pdf</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxJPG">
       <property name="geometry">
        <rect>
         <x>79</x>
         <y>19</y>
         <width>40</width>
         <height>17</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>jpg</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushBSavePlot">
       <property name="geometry">
        <rect>
         <x>144</x>
         <y>6</y>
         <width>58</width>
         <height>26</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>58</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>58</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string>Save Plot</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxPNG">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>19</y>
         <width>45</width>
         <height>17</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>17</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>png</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxBMP">
       <property name="geometry">
        <rect>
         <x>79</x>
         <y>3</y>
         <width>45</width>
         <height>17</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>17</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>bmp</string>
       </property>
      </widget>
     </widget>
     <widget class="QTextBrowser" name="mDataLog">
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="QTextBrowser" name="mMsgsLog">
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBoxPDF</tabstop>
  <tabstop>checkBoxPNG</tabstop>
  <tabstop>checkBoxBMP</tabstop>
  <tabstop>checkBoxJPG</tabstop>
  <tabstop>pushBSavePlot</tabstop>
  <tabstop>pushBPrint</tabstop>
  <tabstop>mDataLog</tabstop>
  <tabstop>mMsgsLog</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
